torch/lib/THD/base/data_channels/DataChannelTCP.cpp

```diff
@@ -70,30 +69,26 @@ DataChannelTCP::DataChannelTCP(InitMethod::Config config, int timeout)
   , _port(0)
   , _timeout(timeout)
   , _poll_events(nullptr)
+  , _processes(config.world_size)
 {
   _rank = config.rank;
 
-  if (_rank == MASTER_RANK) { // MASTER
+  if (_rank == 0) { // MASTER
```

Comment:
For now, we use 0 everywhere in the code so making `MASTER_RANK` in this file makes this more unclear than it should be. We should be consistent. If we decide to change to MASTER_RANK then we should do that everywhere in the code but this is enough work for next PR.